### DISCUSSION TITLE
keybind super+shift+comma executes super+comma

### DISCUSSION THREAD

**Discussion by muramasa64** [ghostty-org/ghostty#3241] [Feature Requests, Ideas] [answered]
keybind super+shift+comma executes super+comma

Version: 1.0.0
Environment: macOS 15.2

I entered super+shift+comma to reload-config. Then, the configuration file opens in textedit. I guess open-reload is executed.

The default keybind for open-reload seems to be super+comma, so it looks like the shift key is ignored.

**Comment by nyngwang** [comment]
I'm not a contributor, but since I have come across your thread, so trying to help.

I cannot reproduce the stated problem on my side (my specs provided below). Given on this, I suspect that it might be related to either:

1. It might be specific to Ghostty `Version: 1.0.0`. You probably need to check their old commits on their defaults.
2. If you're using a custom keyboard, you might want to test that "super-shift-comma" is indeed the sequence sent to Ghostty. (I have no idea how to test this, tho.) On some cases, a custom keyboard might not be able to send certain key-combination.
3. It might be that your `config` was modified accidentally in a way (e.g. other programs, more than one configs, etc) you didn't aware. You might want to try specifying those defaults keybinds explicitly and check for duplication.

Even better, I hope that you might have been solved this problem.

## System Specs

### About this mac

`macOS Sequoia 15.4`;

### Output of `ghostty +version`:

```
Ghostty 1.1.3

Version
  - version: 1.1.3
  - channel: stable
Build Config
  - Zig version: 0.13.0
  - build mode : builtin.OptimizeMode.ReleaseFast
  - app runtime: apprt.Runtime.none
  - font engine: font.main.Backend.coretext
  - renderer   : renderer.Metal
  - libxev     : main.Backend.kqueue
```

**Comment by rhodes-b** [comment, marked as answer]
I'm not sure there is a bug here depending on how you press the keys ghostty may pickup another keybind as you might think you pressed all the keys for example

`keybind = ctrl+shift+h=text:hi`
`keybind = ctrl+h=text:world`

if I don't time shift properly the text `world` will appear on my screen. Ghostty can get inputs very quickly especially depending on os level settings

If you provide an example of the terminal inspector when this happens it might be more indicative if there is actually a bug or not
but since others couldn't reproduce (myself included) and you haven't posted in awhile we will assume your issue is resolved. 
Mitchell has been closing very old issues so I will do the same with this for now feel free to re-open it or make a new discussion if you are still having problems
